docs/index.rst

```diff
@@ -2,41 +2,24 @@
 
 .. _index_penalty:
 
+.. sdk-start-marker
+
+============
+penaltymodel
+============
+
 .. include:: README.rst
```

Comment:
We'll now get a Header1 saying Penaltymodel and under it a header2 saying penaltymodel from README. Would you shift the `.. index-start-marker` in the readme?

Reply:
There isn't one at the top of the `README` currently, so it works

Reply:
![image](https://user-images.githubusercontent.com/34041130/159347008-9fa48007-12ad-490d-ad85-cfc9f0fc48e3.png)

Reply:
oops, you're right, misread